### ISSUE TITLE
how to implement it in Tensorflow

### ISSUE THREAD

**Issue by zhumiao-taylor** [mblondel/soft-dtw#13]
how can I implement the DP recursion in Tensorflow? Tensor does not support assign op.

**Comment by mblondel** [comment]
Tensorflow is not supported at the moment.

**Comment by kjiokeng** [comment]
Hi @mblondel,

Is there anything new concerning the Tensorflow version ? Is it already supported ?

Thanks in advance.

**Comment by AI-Man-gr** [comment]
Hi @mblondel 
I was wondering if Tensorflow is still not supported. I was planning to use it as a loss function. 
Thank you
